plugins/win-dshow/win-dshow.cpp

```diff
@@ -98,9 +98,18 @@ void ffmpeg_log(void *bla, int level, const char *msg, va_list args)
 	DStr str;
 	if (level == AV_LOG_WARNING)
 		dstr_copy(str, "warning: ");
-	else if (level == AV_LOG_ERROR)
+	else if (level == AV_LOG_ERROR) {
+		/* only print first of this message to avoid spam */
+		static bool suppress_app_field_spam = false;
+		if (strcmp(msg, "unable to decode APP fields: Invalid data"
+				" found when processing input\n") == 0) {
+			if (suppress_app_field_spam)
+				return;
+			suppress_app_field_spam = true;
+		}
+
 		dstr_copy(str, "error:   ");
-	else if (level < AV_LOG_ERROR)
+	} else if (level < AV_LOG_ERROR)
```

Comment:
Style: If you change one branch of an `if` to use curly braces, all branches must use curly braces.